rpc/types.go

```diff
@@ -456,3 +456,15 @@ func (ts *Timestamp) UnmarshalJSON(data []byte) error {
 	return nil
 
 }
+
+type OptionsValidateError struct{ Message string }
+
+func (e *OptionsValidateError) ErrorCode() int { return -32003 }
+
+func (e *OptionsValidateError) Error() string { return e.Message }
+
+type KnownAccountsLimitExceededError struct{ Message string }
+
+func (e *KnownAccountsLimitExceededError) ErrorCode() int { return -32005 }
+
+func (e *KnownAccountsLimitExceededError) Error() string { return e.Message }
```

Comment:
please move these to the file where they are used (where SendRawTransactionConditional is implemented), because those are not generic RPC errors

also I suggest to rename OptionsValidateError to TransactionConditionsValidationError

Reply:
[Done](https://github.com/ledgerwatch/erigon/pull/8229/commits/220acfd5b43c35a98c1368938551d6e10f49c773).